python/tk_desktop/desktop_window.py

```diff
@@ -722,6 +725,13 @@ def __launch_app_proxy_for_project(self, project, pipeline_configuration_id=None
             else:
                 raise RuntimeError("unknown platform: %s" % sys.platform)
 
+            if config_path is None:
+                raise RuntimeError("No path set for %s on the Pipeline "
+                                   "Configuration id %d. Please make sure that "
+                                   "it is set on your Shotgun website." %
```

Comment:
Should add a break after the first sentence so that the error fits better in the area we have.
Add in the name of the pipeline configuration in addition to the id so it is easier to find in SG.